nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListFileTransfer.java

```diff
@@ -94,7 +94,7 @@
 
     @Override
     protected String getPath(final ProcessContext context) {
-        return context.getProperty(REMOTE_PATH).getValue();
+        return context.getProperty(REMOTE_PATH).evaluateAttributeExpressions().getValue();
```

Comment:
``ListFileTransfer`` is only used for ``ListFTP`` and ``ListSFTP`` and they do not accept incoming flow files. There is no flow file to use when evaluating the property.